apps/condo/domains/news/components/RecipientCounter.tsx

```diff
@@ -237,11 +238,13 @@ export const RecipientCounter: React.FC<RecipientCounterProps> = ({ newsItemScop
         },
     )
 
+    const throttledGetCounters = useCallback(throttle(getCounters, 1500), [throttle])
```

Comment:
Why `getCounters` not in deps array? To avoid callback rerender?

Reply:
btw, throttle is imported function, its not necessary to put it in reps

Reply:
And why 1500? I'm almost sure we can reproduce error consistently with this value